.github/workflows/update-cudo-catalog.yml

```
@@ -0,0 +1,65 @@
+name: "update-cudo-catalog"
+on:
+  schedule:
+    - cron:  '30 */1 * * *'    # Every 1 hour (as per _PULL_FREQUENCY_HOURS in cudo_catalog.py)
```

This is a bit too frequent for the github action. Could we change it to 7 hours as the other clouds
```suggestion
    - cron:  '30 */7 * * *'    # Every 1 hour (as per _PULL_FREQUENCY_HOURS in cudo_catalog.py)
```